rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/RMServiceImpl.java

```diff
@@ -288,222 +258,146 @@ public String defineNodeSource(final String sessionId, final String nodeSourceNa
-            public InputStream apply(RestClient restClient) {
-                return restClient.getVersion();
-            }
-        });
+        return executeFunctionReturnStreamAsString(RestClient::getVersion);
     }
 
     @Override
-    public String getMBeanInfo(final String sessionId, String name, final List<String> attrs)
+    public String getMBeanInfo(String sessionId, String name, List<String> attrs)
             throws RestServerException, ServiceException {
         try {
             final ObjectName obj = new ObjectName(name);
 
-            return executeFunctionReturnStreamAsString(new Function<RestClient, InputStream>() {
-                @Override
-                public InputStream apply(RestClient restClient) {
-                    return restClient.getMBeanInfo(sessionId, obj, attrs);
-                }
-            });
+            return executeFunctionReturnStreamAsString(restClient -> restClient.getMBeanInfo(sessionId, obj, attrs));
         } catch (MalformedObjectNameException e) {
             throw new ServiceException("Malformed MBean name", e);
         }
     }
 
     @Override
-    public String getNodeMBeanInfo(final String sessionId, final String nodeJmxUrl, final String objectName,
-            final List<String> attrs) throws RestServerException, ServiceException {
-        return executeFunctionReturnStreamAsString(new Function<RestClient, InputStream>() {
-            @Override
-            public InputStream apply(RestClient restClient) {
-                return restClient.getNodeMBeanInfo(sessionId, nodeJmxUrl, objectName, attrs);
-            }
-        });
+    public String getNodeMBeanInfo(String sessionId, String nodeJmxUrl, String objectName, List<String> attrs)
+            throws RestServerException, ServiceException {
+        return executeFunctionReturnStreamAsString(restClient -> restClient.getNodeMBeanInfo(sessionId,
+                                                                                             nodeJmxUrl,
+                                                                                             objectName,
+                                                                                             attrs));
     }
 
     @Override
-    public String getNodeMBeanHistory(final String sessionId, final String nodeJmxUrl, final String objectName,
-            final List<String> attrs, final String timeRange) throws RestServerException, ServiceException {
-        return executeFunctionReturnStreamAsString(new Function<RestClient, InputStream>() {
-            @Override
-            public InputStream apply(RestClient restClient) {
-                return restClient.getNodeMBeanHistory(sessionId, nodeJmxUrl, objectName, attrs, timeRange);
-            }
-        });
+    public String getNodeMBeanHistory(String sessionId, String nodeJmxUrl, String objectName, List<String> attrs,
+            String timeRange) throws RestServerException, ServiceException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)